includes/class-engagement-cloud-loader.php

```diff
@@ -122,5 +122,21 @@ public function run() {
 		foreach ( $this->actions as $hook ) {
 			add_action( $hook['hook'], array( $hook['component'], $hook['callback'] ), $hook['priority'], $hook['accepted_args'] );
 		}
+		
+		$this->enable_a2c();
+	}
+	
+	/**
+	 * Set the options so Api2Cart cam know that modified and created date queries are supported
+	 *
+	 * @since    1.1.0
+	 */
+	private function enable_a2c(){
```

Comment:
This is the only abbreviation of API2Cart. Can we standardise it so all references are or aren't? I vote for aren't.

Reply:
I agree